Classes/author.rb

```diff
@@ -0,0 +1,13 @@
+class Author
+  attr_reader :name, :items
+
+  def initialize(name)
+    @name = name
+    @items = []
```

Comment:
The Author class is not set up according to the UML diagram. Please follow the suggestions to make the Author class with association to the Item class.
```suggestion
  def initialize(first_name, last_name)
    @id = Random.rand(1..1000)
    @first_name = first_name
    @last_name = last_name
    @items = []
```